snapshot/bundlestore/http_store.go

```diff
@@ -10,19 +10,25 @@ import (
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/sethgrid/pester"
 )
 
 func MakeHTTPStore(rootURI string) Store {
 	if !strings.HasSuffix(rootURI, "/") {
 		rootURI = rootURI + "/"
 	}
-	client := &http.Client{Timeout: 30 * time.Second}
+	client := pester.New()
+	client.Backoff = pester.ExponentialBackoff
+	client.MaxRetries = 8
```

Comment:
I think this is functionally the same given that it's an assignment with a clear name. If there were a larger number of options it might make more sense as a global def.